File: app/src/main/java/com/dylag/piotr/e_kartawzorowmatematycznych/MainActivity.java
package com.dylag.piotr.e_kartawzorowmatematycznych;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import static android.provider.AlarmClock.EXTRA_MESSAGE;

public class MainActivity extends AppCompatActivity {


    private Button btnWarBezw, btnPotPier, btnLog, btnSilDwu, btnDwuNew, btnWzSkrMno, btnCiagi, btnFunKwad, btnGeoAna, btnPlan, btnSter, btnTryg, btnKombi, btnRachPraw, btnParDanStat, btnGranCiag, btnPochFun, btnTabWarTryg;
    private TextView textview;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        init();
    }

    public void WartoscBezwzgledna(View view){

        Intent intent = new Intent(this, wartosc_bezwzgledna.class);
       // EditText editText = (EditText) findViewById(R.id.edit_message);
       // String message = editText.getText().toString();
        //intent.putExtra(EXTRA_MESSAGE, message);
        startActivity(intent);

    }


    private void init() {

        btnWarBezw = (Button) findViewById(R.id.btnWartBezw);
        btnPotPier = (Button) findViewById(R.id.btnPotPier);
        btnLog = (Button) findViewById(R.id.btnLog);
        btnSilDwu = (Button)findViewById(R.id.btnSilDwu);
        btnDwuNew = (Button) findViewById(R.id.btnDwuNew);
        btnWzSkrMno = (Button) findViewById(R.id.btnWzSkrMno);
        btnCiagi = (Button) findViewById(R.id.btnCiagi);
        btnFunKwad = (Button) findViewById(R.id.btnFunKwad);
        btnGeoAna = (Button) findViewById(R.id.btnGeoAna);
        btnPlan = (Button) findViewById(R.id.btnPlan);
        btnSter = (Button) findViewById(R.id.btnSter);
        btnTryg = (Button) findViewById(R.id.btnTryg);
        btnKombi = (Button) findViewById(R.id.btnKombi);
        btnRachPraw = (Button) findViewById(R.id.btnRachPraw);
        btnParDanStat = (Button) findViewById(R.id.btnParDanStat);
        btnGranCiag = (Button)findViewById(R.id.btnGranCiag);
        btnPochFun = (Button) findViewById(R.id.btnPochFun);
        btnTabWarTryg = (Button) findViewById(R.id.btnTabWarTryg);


    }

    public void PotegiPierwiastki(View view) {

        Intent intentpier = new Intent(this, PotegiPierwiastki.class);
        startActivity(intentpier);
    }

    public void logarytmy(View view) {

        Intent intentlog = new Intent(this, Logarytmy.class);
        startActivity(intentlog);
    }

    public void SilniaDwumian(View view) {

        Intent intentsilniadwumian = new Intent(this, SilniaDwumian.class);
        startActivity(intentsilniadwumian);
    }

    public void DwumianNew(View view) {
        Intent intentdwumiannew = new Intent(this, DwumianNew.class);
        startActivity(intentdwumiannew);
    }

    public void WzorySkrMnoz(View view) {
        Intent intentwzoryskrmnoz = new Intent(this, WzorySkrMnoz.class);
        startActivity(intentwzoryskrmnoz);
    }


    public void ciagi(View view) {
        Intent intentciagi = new Intent(this, Ciagi.class);
        startActivity(intentciagi);
    }

    public void FunKwad(View view) {

        Intent intentfunkwad = new Intent(this, FunKwad.class);
        startActivity(intentfunkwad);
    }

    public void GeoAna(View view) {

        Intent intentgeoana = new Intent(this, GeoAna.class);
        startActivity(intentgeoana);
    }

    public void Plan(View view) {

        Intent intentplan = new Intent(this, Plan.class);
        startActivity(intentplan);
    }

    public void Ster(View view) {

        Intent intentster = new Intent(this, Ster.class);
        startActivity(intentster);
    }

    public void Tryg(View view) {

        Intent intenttryg = new Intent(this, Tryg.class);
        startActivity(intenttryg);
    }

    public void Kombi(View view) {

        Intent intentkombi = new Intent(this, Kombi.class);
        startActivity(intentkombi);

    }


    public void RachPraw(View view) {

        Intent intentrachpraw = new Intent(this, Rachpraw.class);
        startActivity(intentrachpraw);
    }

    public void ParDanStat(View view) {

        Intent intentpardanstat = new Intent(this, ParDanStat.class);
        startActivity(intentpardanstat);
    }

    public void GranCiag(View view) {

        Intent intentgranciag = new Intent(this, GranCiag.class);
        startActivity(intentgranciag);
    }

    public void PochFun(View view) {

        Intent intentpochfun = new Intent(this, PochFun.class);
        startActivity(intentpochfun);
    }

    public void TabWarFunTryg(View view) {

        Intent intenttabwarfuntryg = new Intent(this, TabWarFunTryg.class);
        startActivity(intenttabwarfuntryg);
    }
}

File: app/src/main/java/com/dylag/piotr/e_kartawzorowmatematycznych/MainActivity.kt
package com.dylag.piotr.e_kartawzorowmatematycznych

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

import android.provider.AlarmClock.EXTRA_MESSAGE

class MainActivity : AppCompatActivity() {


    private var btnWarBezw: Button? = null
    private var btnPotPier: Button? = null
    private var btnLog: Button? = null
    private var btnSilDwu: Button? = null
    private var btnDwuNew: Button? = null
    private var btnWzSkrMno: Button? = null
    private var btnCiagi: Button? = null
    private var btnFunKwad: Button? = null
    private var btnGeoAna: Button? = null
    private var btnPlan: Button? = null
    private var btnSter: Button? = null
    private var btnTryg: Button? = null
    private var btnKombi: Button? = null
    private var btnRachPraw: Button? = null
    private var btnParDanStat: Button? = null
    private var btnGranCiag: Button? = null
    private var btnPochFun: Button? = null
    private var btnTabWarTryg: Button? = null
    private val textview: TextView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        init()
    }

    fun wartoscBezwzgledna(view: View) {

        val intent = Intent(this, wartosc_bezwzgledna::class.java)
        startActivity(intent)

    }


    private fun init() {

        btnWarBezw = findViewById(R.id.btnWartBezw) as Button
        btnPotPier = findViewById(R.id.btnPotPier) as Button
        btnLog = findViewById(R.id.btnLog) as Button
        btnSilDwu = findViewById(R.id.btnSilDwu) as Button
        btnDwuNew = findViewById(R.id.btnDwuNew) as Button
        btnWzSkrMno = findViewById(R.id.btnWzSkrMno) as Button
        btnCiagi = findViewById(R.id.btnCiagi) as Button
        btnFunKwad = findViewById(R.id.btnFunKwad) as Button
        btnGeoAna = findViewById(R.id.btnGeoAna) as Button
        btnPlan = findViewById(R.id.btnPlan) as Button
        btnSter = findViewById(R.id.btnSter) as Button
        btnTryg = findViewById(R.id.btnTryg) as Button
        btnKombi = findViewById(R.id.btnKombi) as Button
        btnRachPraw = findViewById(R.id.btnRachPraw) as Button
        btnParDanStat = findViewById(R.id.btnParDanStat) as Button
        btnGranCiag = findViewById(R.id.btnGranCiag) as Button
        btnPochFun = findViewById(R.id.btnPochFun) as Button
        btnTabWarTryg = findViewById(R.id.btnTabWarTryg) as Button


    }

    fun potegiPierwiastki(view: View) {

        val intentpier = Intent(this, PotegiPierwiastki::class.java)
        startActivity(intentpier)
    }

    fun logarytmy(view: View) {

        val intentlog = Intent(this, Logarytmy::class.java)
        startActivity(intentlog)
    }

    fun SilniaDwumian(view: View) {

        val intentsilniadwumian = Intent(this, SilniaDwumian::class.java)
        startActivity(intentsilniadwumian)
    }

    fun DwumianNew(view: View) {
        val intentdwumiannew = Intent(this, DwumianNew::class.java)
        startActivity(intentdwumiannew)
    }

    fun WzorySkrMnoz(view: View) {
        val intentwzoryskrmnoz = Intent(this, WzorySkrMnoz::class.java)
        startActivity(intentwzoryskrmnoz)
    }


    fun ciagi(view: View) {
        val intentciagi = Intent(this, Ciagi::class.java)
        startActivity(intentciagi)
    }

    fun FunKwad(view: View) {

        val intentfunkwad = Intent(this, FunKwad::class.java)
        startActivity(intentfunkwad)
    }

    fun GeoAna(view: View) {

        val intentgeoana = Intent(this, GeoAna::class.java)
        startActivity(intentgeoana)
    }

    fun Plan(view: View) {

        val intentplan = Intent(this, Plan::class.java)
        startActivity(intentplan)
    }

    fun Ster(view: View) {

        val intentster = Intent(this, Ster::class.java)
        startActivity(intentster)
    }

    fun Tryg(view: View) {

        val intenttryg = Intent(this, Tryg::class.java)
        startActivity(intenttryg)
    }

    fun Kombi(view: View) {

        val intentkombi = Intent(this, Kombi::class.java)
        startActivity(intentkombi)

    }


    fun RachPraw(view: View) {

        val intentrachpraw = Intent(this, Rachpraw::class.java)
        startActivity(intentrachpraw)
    }

    fun ParDanStat(view: View) {

        val intentpardanstat = Intent(this, ParDanStat::class.java)
        startActivity(intentpardanstat)
    }

    fun GranCiag(view: View) {

        val intentgranciag = Intent(this, GranCiag::class.java)
        startActivity(intentgranciag)
    }

    fun PochFun(view: View) {

        val intentpochfun = Intent(this, PochFun::class.java)
        startActivity(intentpochfun)
    }

    fun TabWarFunTryg(view: View) {

        val intenttabwarfuntryg = Intent(this, TabWarFunTryg::class.java)
        startActivity(intenttabwarfuntryg)
    }
}
